.github/workflows/performance-tests.yml

```
@@ -0,0 +1,404 @@
+        run: |
+          cd frontend
+          npm run analyze:bundle
+          node scripts/check-bundle-size.js
+
+  database-performance:
+    runs-on: ubuntu-latest
+    if: github.event.inputs.test_type == 'all' || github.event.inputs.test_type == 'database' || github.event.inputs.test_type == ''
+    
+    services:
+      postgres:
+        image: postgres:${{ env.POSTGRES_VERSION }}
```

actionlint (1.7.11)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)